docs/user-guides/getting-started.md

```diff
@@ -1,33 +1,30 @@
-# emqx-operator
+**Note**: EMQ X Operator requires use of Kubernetes v1.20.0 and up.
```

Comment:
EMQ X Operator requires use of Kubernetes v1.20.0 and up ->
EMQ X Operator requires Kubernetes v1.20.0 and up